extensions/dub-link-shortener/README.md

```diff
@@ -3,7 +3,3 @@
 Dub is the open-source link management platform for modern marketing teams to create marketing campaigns, link sharing features, and referral programs.
 
 This extension allows you to shorten and retrieve your Dub links directly inside Raycast.
-
-# Instructions
-
-You will need the API key from Dub. To generate API key, follow [these instructions](https://dub.co/docs/api-reference/tokens).
```

Comment:
Suggestion (non-blocking): Might want to add OAuth instructions (and removing earlier workspace key) + showcase end-2-end flow like this: https://www.raycast.com/koinzhang/github-gist

Reply:
Sounds like a good idea, maybe it could be good to do that before merge?

Reply:
@devkiran here is the gif you can save into `media/` dir for the e2e flow in readme:

![2024-07-31 02 24 31](https://github.com/user-attachments/assets/8ec09bd9-e03e-45ec-bdfa-b95440740137)

Reply:
@jfkisafk Sounds good. I've updated the README. Thank you